template/template.go

```diff
@@ -317,7 +318,7 @@ func (te Expander) ExpandHTML(templateFiles []string) (result string, resultErr
 			var ok bool
 			resultErr, ok = r.(error)
 			if !ok {
-				resultErr = fmt.Errorf("panic expanding template %v: %v", te.name, r)
+				resultErr = errors.Wrapf(r.(error), "panic expanding template %v", te.name)
```

Comment:
This branch is reached if `r` couldn't be coerced to an `error` type, so doing `r.(error)` here would fail. In the case where it's not an `error`, we can just print it via `%v` I believe.

Reply:
Whoops thanks for catching that :)